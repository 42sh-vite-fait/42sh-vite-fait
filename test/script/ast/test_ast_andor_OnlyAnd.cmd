(file name|grep type) 2>&1 >/dev/null&&cat name&&rm name
