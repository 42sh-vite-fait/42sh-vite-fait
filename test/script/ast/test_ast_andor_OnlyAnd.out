AST:
AND_OR: &&
	AND_OR: &&
		SUBSHELL: ()
			redirections#0: IO=2, Operator=GREATAND, Word=1
			redirections#1: IO=1, Operator=GREAT, Word=/dev/null
			PIPE_SEQUENCE: |
				SIMPLE_COMMAND: file
					args#1: name
				SIMPLE_COMMAND: grep
					args#1: type
		SIMPLE_COMMAND: cat
			args#1: name
	SIMPLE_COMMAND: rm
		args#1: name
